articles/service-bus-messaging/advanced-features-overview.md

```diff
@@ -33,7 +33,7 @@ A queue or subscription client can defer retrieval of a received message until a
 A transaction groups two or more operations together into an execution scope. Service Bus allows you to group operations against multiple messaging entities within the scope of a single transaction. A message entity can be a queue, topic, or subscription. For more information, see [Overview of Service Bus transaction processing](service-bus-transactions.md).
 
 ## Autodelete on idle
-Autodelete on idle enables you to specify an idle interval after which a queue or topic subscription is automatically deleted. The interval is reset when there is traffic on the entity. The minimum duration is 5 minutes. 
+Autodelete on idle enables you to specify an idle interval after which a queue or topic subscription is automatically deleted. The interval is reset when a message is added to or removed from the subscription. The minimum duration is 5 minutes. 
```

Comment:
Is it possible to get a specific list of actions that count as traffic? E.g.:
- locking the message?
- unlocking the message?
- message auto-deleted when TTL expires?

Reply:
Thanks @gukoff